src/services/member/controller.ts

```diff
@@ -51,6 +59,33 @@ const controller: FastifyPluginAsync = async (fastify) => {
     },
   );
 
+  // get current member storage files metadata
+  fastify.get<{ Querystring: Pagination }>(
+    '/current/storage/files',
+    { schema: getStorageFiles, preHandler: isAuthenticated },
+    async ({ user, query: { page, pageSize } }, reply) => {
+      if (
+        page < FILE_METADATA_MIN_PAGE ||
+        pageSize < FILE_METADATA_MIN_PAGE_SIZE ||
+        pageSize > FILE_METADATA_MAX_PAGE_SIZE
+      ) {
+        reply.status(StatusCodes.BAD_REQUEST).send();
+        return;
+      }
+      const member = notUndefined(user?.member);
+      const storageFilesMetadata = await storageService.getStorageFilesMetadata(
+        member,
+        fileService.fileType,
+        { page, pageSize },
+        buildRepositories(),
+      );
```

Comment:
I usually try to keep `actor` as first arg, then `repositories` as 2nd arg for some kind of consistency.